soroban-settings/phase1.json

```diff
@@ -21,31 +21,31 @@
         "tx_max_read_bytes": 133120,
         "tx_max_write_ledger_entries": 20,
         "tx_max_write_bytes": 66560,
-        "fee_read_ledger_entry": 1000,
-        "fee_write_ledger_entry": 3000,
-        "fee_read1_kb": 1000,
-        "bucket_list_target_size_bytes": 14495514624,
-        "write_fee1_kb_bucket_list_low": 1000,
-        "write_fee1_kb_bucket_list_high": 4000000,
+        "fee_read_ledger_entry": 6250,
+        "fee_write_ledger_entry": 10000,
+        "fee_read1_kb": 1786,
+        "bucket_list_target_size_bytes": 13958643712,
```

Comment:
Where does this number come from? It's ~13.9 GB. The doc suggests 13 GB; I think we should double-check the current ledger size before the upgrade, but in any case this value seems weird.

Reply:
That's the number of bytes in 13 GB right? Using 1 GB == 1,073,741,824 bytes. I can update this to 13000000000 if that's what was used for your calculations.

Reply:
Hmm, we actually need to check which unit has been used for the 'current' size computations so far, as it actually makes a huge difference at this scale.

Reply:
Yeah most of the size values use 1KB == 1024 in this file.